### PR TITLE
Feature inform signaturealgorithm

### PR DESCRIPTION
**Pull request by JoseTM**
default algorithm seem to be removed  from xml-crypto; so now must it must be specified. WSSecurityCert is not setting singnatureAlgorithm actually so just rsa-sha256 is been used. I tried to fix this behaviour to be able of setting signature algorithm.

### CODE REVIEW

File: test/security/WSSecurityCert.js
Comment (w666) [inline test/security/WSSecurityCert.js:1]: Unfortunately, it is very hard to review changes in this file. I agree that code formatting is not the best in this project, but I don't think it should be addressed in this PR. Could you please revert code-reformatting and leave just actual test changes.

Comment (JoseTM) [reply test/security/WSSecurityCert.js:1]: Yes,  of course. And I apollogize

**Review by w666 — APPROVED** [review]

**Comment by w666** [comment]
@JoseTM let me know if you need this feature to be released asap, if no I will do at my normal release schedule in 1-2 weeks.

**Comment by JoseTM** [comment]
In your normal workflow is fine. Pleased to contribute and thanks for your job

**Comment by SzymonNadbrzezny** [comment]
Good to know it's going to be released soon. :D

**Comment by w666** [comment]
1.1.4 has been released

**Pull request merged**